cdap-proto/src/main/java/co/cask/cdap/proto/ProgramRunStatus.java

```diff
@@ -53,4 +53,14 @@ public static NodeStatus toNodeStatus(ProgramRunStatus status) {
                                                          status.name()));
     }
   }
+
+  public static ProgramStatus toProgramStatus(ProgramRunStatus status) {
+    switch(status) {
+      case STARTING:
```

Comment:
what about RESUMED? SUSPENDED?
Should either of those result in a RUNNING status?
@sagarkapare ?